### ISSUE TITLE
Google Sheets CMS does not update realtime responses

### ISSUE THREAD

**Issue by SweetBabySoylentGithub** [jovotech/jovo-framework#394]
## I'm submitting a...

- [x] Bug report
- [ ] Feature request
- [ ] Documentation issue or request
- [ ] Other... Please describe:

## Expected Behavior

Alexa / Google should read responses directly from Google Sheets CMS.
Google Sheets CMS should update Alexa / Google Responses the instant a sheet is modified.

## Current Behavior

Alexa / Google responses are not updated until Jovo is deployed to AWS Lambda
Google Sheets CMS does not update Alexa / Google Responses until Jovo Project is re-deployed.

### Error log
```
If you have an error log, please paste it here.
We spent 1,875 pipeline minutes in the past 2 weeks JUST to update Alexa / Google responses with Google Sheets CMS
```
![image](https://user-images.githubusercontent.com/39890315/53274913-10d07700-36c7-11e9-9833-9cc9d4acd0ec.png)

## Your Environment

* Jovo Framework version used: 2.0.15
* jovo-cms-googlesheets: 2.0.14
* Operating System: Gitlab Pipelines + Docker

**Comment by mderrien29** [comment]
Well, checking the Gsheet everytime would be slow, right ?

IMO, a plugin that periodically update i18n files from google Sheets (and keep a save somewhere in case the Sheet goes down) would be the best of both worlds.

**Comment by jankoenig** [comment]
We thought about adding a configuration that determines how often the CMS should be called:
* `app`: Stored in the `app` object when it is started (e.g. when doing `jovo run` or when the Lambda is starting up). This is the default option and useful for production app.
* `session`: Content gets retrieved for every new session. Will potentially cause longer load times for `LAUNCH` requests, but after that stay fast.
* `request`: Content gets retrieved with every request. Slowest, but most up to date option.


One more note: Right now, it's not necessary to redeploy to Lambda, but rather to wait until the current "warm" Lambda is turned off again.

**Comment by crazzywizard** [comment]
How can we change it from ```app``` to ```0n new session```

**Comment by jankoenig** [comment]
Hi @crazzywizard, currently, this isn't possible. We're thinking about adding the functionality

**Comment by jankoenig** [comment]
This was just implemented with #428 of the latest release: https://github.com/jovotech/jovo-framework/blob/master/CHANGELOG.md#214-2019-03-22. You can update with `jovo update`.

In your Google Sheets config in the `config.js`, you can add `caching: false,` which should stop caching and let your app retrieve CMS content with every request. Docs are coming soon.
